AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLDocument/Query/GraphQLDocument+ListQuery.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2018-2019 Amazon.com,
+// Copyright 2018-2020 Amazon.com,
```

Comment:
No good way to comment on L14: Should this class actually be public? Wouldn't the customer be doing a request based or mode-based query rather than creating documents? Or is it public so the DataStore category can create documents directly?